tests/src/JIT/opt/Devirtualization/enum.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+enum MyEnum { One, Two, Three, Four, Five, Six, Seven, Eight, Nine, Ten };
+
+// ToString should be devirtualizable
+
+public class Test
+{
+    public static int Main()
+    {
+        string s = (MyEnum.Seven).ToString();
+        return (s.Length == "Seven".Length ? 100 : -1);
```

Comment:
All I meant was that I took the code snippet from #1166 and used it directly without thinking much about the second line, since the first line was the interesting part.

I updated the second line to force use of the virtual Equals so it's now interesting too.